Algorithm/CMakeLists.txt

```diff
@@ -10,51 +10,64 @@
 
 o2_add_header_only_library(Algorithm INTERFACE_LINK_LIBRARIES O2::Headers)
 
-o2_target_man_page(Algorithm NAME Algorithm SECTION 3)
-o2_target_man_page(Algorithm NAME algorithm_parser SECTION 3)
-
-o2_add_test(o2formatparser
-            SOURCES test/o2formatparser.cxx
-            COMPONENT_NAME Algorithm
-            PUBLIC_LINK_LIBRARIES O2::Algorithm
-            LABELS algorithm)
-
-o2_add_test(headerstack
-            SOURCES test/headerstack.cxx
-            COMPONENT_NAME Algorithm
-            PUBLIC_LINK_LIBRARIES O2::Algorithm
-            LABELS algorithm)
-
-o2_add_test(parser
-            SOURCES test/parser.cxx
-            COMPONENT_NAME Algorithm
-            PUBLIC_LINK_LIBRARIES O2::Algorithm
-            LABELS algorithm)
-
-o2_add_test(tableview
-            SOURCES test/tableview.cxx
-            COMPONENT_NAME Algorithm
-            PUBLIC_LINK_LIBRARIES O2::Algorithm
-            LABELS algorithm)
-
-o2_add_test(pageparser
-            SOURCES test/pageparser.cxx
-            COMPONENT_NAME Algorithm
-            PUBLIC_LINK_LIBRARIES O2::Algorithm
-            LABELS algorithm)
-
-o2_add_test(mpl_tools
-            SOURCES test/test_mpl_tools.cxx
-            COMPONENT_NAME Algorithm
-            LABELS algorithm)
-
-o2_add_test(RangeTokenizer
-            SOURCES test/test_RangeTokenizer.cxx
-            COMPONENT_NAME Algorithm
-            LABELS algorithm)
-
-
-o2_add_test(FlattenRestore
-            SOURCES test/test_FlattenRestore.cxx
-            COMPONENT_NAME Algorithm
-            LABELS algorithm)
+o2_target_man_page(
+  Algorithm
+  NAME Algorithm
+  SECTION 3)
```

Comment:
I think the old indentation is still more readable and since the coding convention say that lines up to 120 chars are allowed, what are we trying to fix with this, exactly?

Reply:
I suppose you are referring to the [C++ formatting rules](https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=Line_Length#Line_Length). Those are enforced by `clang-format`.
I am not aware of any document that would describe CMake formatting rules besides `.cmake-format.py` which defines `line_width = 80`

Reply:
80 would also be fine, but there you have 20...

To me:

```
o2_target_man_page(Algorithm NAME Algorithm SECTION 3)
```

looks infinitely better, frankly. 

IMHO, it's fine to break long lines or to enforce it for things like SOURCES when there is more than a couple of files, but doing everywhere indiscriminately reduces readability.

Reply:
totally agree, I would keep it as 1-liner where possible. I would only cut ridiculously long lines, (certainly not 80, it is 2021 with 5k screens :)), or some special selected arguments like `SOURCES` if there is more than 1 parameter for that argument.